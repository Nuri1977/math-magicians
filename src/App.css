.calculator {
  text-align: center;
  width: 240px;
  margin: auto;
}
