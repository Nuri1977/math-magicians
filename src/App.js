import React, { Component } from 'react';
import './App.css';
import Calculator from './components/Calculator';

class App extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return (
      <div className="Calculator">
        <Calculator />
      </div>
    );
  }
}

export default App;
