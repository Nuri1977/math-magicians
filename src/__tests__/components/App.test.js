import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import App from '../../App';

const MockApp = () => (
  <BrowserRouter>
    <App />
  </BrowserRouter>
);

describe('Testing App component', () => {
  it('App renders correctly', () => {
    const tree = renderer
      .create(<MockApp />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
