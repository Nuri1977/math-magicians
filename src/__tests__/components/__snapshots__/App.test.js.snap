// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App component App renders correctly 1`] = `
<div>
  <header>
    <div
      className="container flex"
    >
      <h1>
        <a
          className="logo"
          href="/"
          onClick={[Function]}
        >
          Math Magicians
        </a>
      </h1>
      <nav>
        <ul
          className="flex"
        >
          <li>
            <a
              href="/"
              onClick={[Function]}
            >
              Home
            </a>
          </li>
          <li>
            <a
              href="/calculator"
              onClick={[Function]}
            >
              Calculator
            </a>
          </li>
          <li>
            <a
              href="/quote"
              onClick={[Function]}
            >
              Quote
            </a>
          </li>
        </ul>
      </nav>
    </div>
  </header>
  <div
    className="container"
  >
    <h1
      className="home-title"
    >
      Welcome to our page!
    </h1>
    <p
      className="home-pg"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Libero voluptatibus, recusandae, temporibus dignissimos ullam porro earum repellendus explicabo esse officiis ratione nulla vero aliquam nisi molestias placeat, duci mus quos nobis.
    </p>
    <p
      className="home-pg"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Libero voluptatibus, recusandae, temporibus dignissimos ullam porro earum repellendus explicabo esse officiis ratione nulla vero aliquam nisi molestias placeat, duci mus quos nobis.
    </p>
  </div>
</div>
`;
